src/Part2.js

```diff
@@ -0,0 +1,133 @@
+import axios from "axios";
+
+
+const sectionCountries = document.getElementById('country-information');
+let errorMessage = "";
+
+
+async function fetchDataSearch(countryName) {
+
+    try {
+
+        const dataCountry = await axios.get(`https://restcountries.com/v2/name/${countryName}`);
+
+
+        // Leeg de lijst
+        sectionCountries.replaceChildren();
+        errorMessage.replaceChildren();
+
+
+
+        // Map door de data heen
+        dataCountry.data.map( ( country ) => {
+
+            //maak variable aan uit arrays
+            const {flags:{png}, name, capital, subregion, currencies, population, languages} = country;
+
+
+            //make array list of needed names.
+            let coinString;
+            const coinArray = currencies.map((coin) => {
+                return coin.name;
+            })
+
+            // print strings
+            if (currencies.length === 1) {
+                coinString = `you can pay with ${coinArray[0]}`;
+            }else {
+                coinString = `you can pay with ${coinArray[0]} and ${coinArray[1]}`;
+            }
+
+            let languagesString;
+            const languagesArray= languages.map((lan) => {
+                return lan.name;
+            })
+
+               if (languagesArray.length === 1) {
+                   languagesString = `they speak ${languagesArray[0]}`;
+               } else if (languages.length === 2) {
+                       languagesString = `they speak ${languagesArray[0]} and ${languagesArray[1]}`;
+               }else {
+                       languagesString = `they speak ${languagesArray[0]}, ${languagesArray[1]} and ${languagesArray[2]}`;
+               }
```

Comment:
Dit is niet echt uitbreidbaar als er meerdere languages zouden komen. Je kan hier gebruikmaken van een for-loop of de [Array.join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)